src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/Hoster.java

```diff
@@ -251,6 +251,9 @@ boolean forkGitHub(String owner, String repo, String newName, List<String> maint
     private static void setupRepository(GHRepository r, boolean useGHIssues) throws IOException {
         r.enableIssueTracker(useGHIssues);
         r.enableWiki(false);
+        r.setHomepage("https://plugins.jenkins.io/" + r.getName().replace("-plugin", "") + "/");
+        r.setEmailServiceHook("jenkinsci-commits@googlegroups.com");
```

Comment:
Apparently this is very common for many plugins and core repositories, and therefore I'd propose to enable it for new plugins too, in case someone uses the mailing list for reading purposes.

Reply:
I think this is an old thing and should be done in a separate PR and checked if its useful